cmake/Modules/Platform/Emscripten.cmake

```diff
@@ -95,6 +95,7 @@ set(CMAKE_C_COMPILER_AR "${CMAKE_AR}")
 set(CMAKE_CXX_COMPILER_AR "${CMAKE_AR}")
 set(CMAKE_C_COMPILER_RANLIB "${CMAKE_RANLIB}")
 set(CMAKE_CXX_COMPILER_RANLIB "${CMAKE_RANLIB}")
+set(CMAKE_NM "${EMSCRIPTEN_ROOT_PATH}/emnm${EMCC_SUFFIX}")
```

Comment:
Can you move this up to line 92?  (too keep all the EMSCRIPTEN_ROOT_PATH usages together?)

Reply:
Of course :)